### DISCUSSION TITLE
Request for steps to build qdrant for arm.

### DISCUSSION THREAD

**Discussion by cravishe** [qdrant/qdrant#3377] [Q&A]
Hello,
I'm exploring the way to cross compile qdrant for arm64 architecture. Are there any steps for this. Kindly help me with this information.
Thank you

**Comment by timvisee** [comment]
While we don't have instructions for this ready (because we don't do this directly), we do cross compile our Docker image for ARM64 somewhere in our pipeline.

If building a Docker image is sufficient, this may help: https://github.com/qdrant/qdrant/blob/c6a351c82d57eb786b75dac785de7b085c6f80ff/.github/workflows/docker-image.yml#L25-L55

Otherwise I recommend to look up more general guides for cross compilation in Rust.

**Reply by cravishe** [reply]
Thanks for the reply. I'm not planning to build docker image. I want to build from source for arm 64.  I'll check more on cross compilation in Rust. If any other helpful guide or information is there on this, kindly do let me know.
